src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -74,15 +74,15 @@ type ListProps = {
   isSearching: boolean
   canAddAmount: boolean
 }
-
 const List = ({fungibilityFilter, isSearching, canAddAmount}: ListProps) => {
-  const isNftListVisible = fungibilityFilter === 'nft' && !isSearching
+  const showOnlyNftList = fungibilityFilter === 'nft' && !isSearching
```

Comment:
refactored "isVisibile" gives the idea that it can me displayed along with another components